data/os/Ubuntu/22.04.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Can you please raise a new PR that adds Ubuntu 22.04 support? That's a new feature, unrelated to the YAML format support. We require those different PRs because we autogenerate the changelog based on PR titles.